src/core_ocean/shared/mpas_ocn_high_freq_thickness_hmix_del2.F

```diff
@@ -63,98 +62,80 @@ module ocn_high_freq_thickness_hmix_del2
 
       real (kind=RKIND) :: invAreaCell, hhf_turb_flux, flux, r_tmp
 
       real (kind=RKIND), dimension(:), pointer :: areaCell, dvEdge, dcEdge
       real (kind=RKIND), dimension(:), pointer :: meshScalingDel2
 
-      err = 0
-
-      if(.not.config_use_highFreqThick_del2) return
+      ! End preamble
+      !-------------
+      ! Begin code
 
+      ! Set default error flag and reeturn if not selected
+      ! Otherwise, start timer and continue
+      err = 0
+      if(.not. config_use_highFreqThick_del2) return
       call mpas_timer_start("thick hmix del2")
 
-      call mpas_pool_get_dimension(meshPool, 'nCells', nCells)
-      call mpas_pool_get_dimension(meshPool, 'nVertLevels', nVertLevels)
-
-      call mpas_pool_get_array(meshPool, 'maxLevelEdgeTop', maxLevelEdgeTop)
-      call mpas_pool_get_array(meshPool, 'cellsOnEdge', cellsOnEdge)
-      call mpas_pool_get_array(meshPool, 'edgeMask', edgeMask)
-      call mpas_pool_get_array(meshPool, 'areaCell', areaCell)
-      call mpas_pool_get_array(meshPool, 'dvEdge', dvEdge)
-      call mpas_pool_get_array(meshPool, 'dcEdge', dcEdge)
-      call mpas_pool_get_array(meshPool, 'meshScalingDel2', meshScalingDel2)
-
-      call mpas_pool_get_array(meshPool, 'nEdgesOnCell', nEdgesOnCell)
-      call mpas_pool_get_array(meshPool, 'edgesOnCell', edgesOnCell)
-      call mpas_pool_get_array(meshPool, 'edgeSignOnCell', edgeSignOnCell)
-
       !$omp parallel
-      !$omp do schedule(runtime) private(invAreaCell, i, iEdge, cell1, cell2, r_tmp, k, hhf_turb_flux, flux)
-      do iCell = 1, nCells
+      !$omp do schedule(runtime) &
+      !$omp    private(i, k, iEdge, cell1, cell2, invAreaCell, &
+      !$omp            r_tmp, hhf_turb_flux, flux)
+      do iCell = 1, nCellsAll
          invAreaCell = 1.0_RKIND / areaCell(iCell)
          do i = 1, nEdgesOncell(iCell)
             iEdge = edgesOnCell(i, iCell)
             cell1 = cellsOnEdge(1,iEdge)
             cell2 = cellsOnEdge(2,iEdge)
 
-            r_tmp = meshScalingDel2(iEdge) * config_highFreqThick_del2 * dvEdge(iEdge) / dcEdge(iEdge)
+            r_tmp = meshScalingDel2(iEdge)*config_highFreqThick_del2* &
+                    dvEdge(iEdge)/dcEdge(iEdge)
 
             do k = 1, maxLevelEdgeTop(iEdge)
                ! \nabla h^{hf} on edge
-               hhf_turb_flux = highFreqThickness(k,cell2) - highFreqThickness(k,cell1)
+               hhf_turb_flux = highFreqThickness(k,cell2) - &
+                               highFreqThickness(k,cell1)
 
                ! div(\kappa_{hf} \nabla h^{hf}) at cell center
-               flux = hhf_turb_flux * edgeMask(k, iEdge) * r_tmp
+               flux = hhf_turb_flux*edgeMask(k, iEdge)*r_tmp
 
-               tend_highFreqThickness(k, iCell) = tend_highFreqThickness(k, iCell) - edgeSignOnCell(i, iCell) * flux * invAreaCell
+               tend_highFreqThickness(k, iCell) = &
+               tend_highFreqThickness(k, iCell) - &
+                      edgeSignOnCell(i,iCell)*flux*invAreaCell
```

Comment:
@philipwjones I find this formatting hard to read, as a line continuation is expected to be indented, and I think operators are easier to read at the beginning than at the end. Is this from a standard, or your own preference?

Unfortunately Fortran does not have a formatting standard like python PEP8, and we never made a standard ourselves. We should do that. Until then, I would prefer to revert some of these unusual lines. Removing spaces around * is fine though.

Reply:
@mark-petersen no standard, but having operators at the end guarantees an error on the first line if you forget the continuation character.  I don't think I've been that consistent on that front so it's not a big deal either way. And I tend to like lining thing up column-wise which was the reason for the above alignment. But I don't mind changing formats as long as the lines are shorter than ~75 characters so they'll fit in a typical 80-character terminal window, even with line numbers on.

Reply:
Yes, I can see the appeal of lining up the indices. Let's leave your spacing in this PR. I think Fortran spacing is a larger conversation so we can unify the format as we make changes in the future.